loadbalancer/traefik.toml

```diff
@@ -4,18 +4,23 @@ defaultEntryPoints = ["http", "https"]
 email = "tech@sse.rit.edu"
 storage = "/etc/traefik/acme/acme.json"
 entryPoint = "https"
-delayDontCheckDNS = 0
 acmeLogging = true
-OnHostRule = true
+onHostRule = true
+
+[acme.dnsChallenge]
+  delayBeforeCheck = 0
+
+[acme.httpChallenge]
+  entryPoint = "http"
 
 [entryPoints]
-    [entryPoints.http]
-    address = ":80"
-      [entryPoints.http.redirect]
-        entryPoint = "https"
-    [entryPoints.https]
-    address = ":443"
-    	[entryPoints.https.tls]
```

Comment:
Why the whitespace changes?

Reply:
They were 4 spaces. Everything else was 2 spaces. Consistency, man.